lips/lip-6.mdx

```diff
@@ -0,0 +1,253 @@
+  * Parameters:
+    * `validator_account: address` - validator's account address
+    * `consensus_pubkey: vector<u8>` - consensus public key
+    * `validator_network_addresses: vector<u8>` - validator's network address
+    * `fullnode_network_addresses: vector<u8>` - validator's full node network address
+  * Events: none
+  * Errors:
+    * `validator_account` is not a validator: `NOT_PUBLISHED.ValidatorConfig::EVALIDATOR_CONFIG`
+    * Sender is not validator's operator: `INVALID_ARGUMENT.ValidatorConfig::EINVALID_TRANSACTION_SENDER`
+    * Invalid consensus public key: `INVALID_ARGUMENT.ValidatorConfig::EINVALID_CONSENSUS_KEY`
+* [`set_validator_config_and_reconfigure`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/set_validator_config_and_reconfigure.move) - to change values of the Validator Config resource and update the config in the Validator Set (aborts if the Validator Owner is not in the Validator Set)
+  * Parameters:
+    * `validator_account: address` - validator's account address
+    * `consensus_pubkey: vector<u8>` - consensus public key
+    * `validator_network_addresses: vector<u8>` - validator's network address
+    * `fullnode_network_addresses: vector<u8>` - validator's full node network address
+  * Events: NewEpochEvent
+  * Errors:
+    * `validator_account` is not a validator: `NOT_PUBLISHED.ValidatorConfig::EVALIDATOR_CONFIG`
+    * Sender is not validator's operator: `INVALID_ARGUMENT.ValidatorConfig::EINVALID_TRANSACTION_SENDER`
+    * Invalid consensus public key: `INVALID_ARGUMENT.ValidatorConfig::EINVALID_CONSENSUS_KEY`
+    * `validator_account` is not in the ValidatorSet: `INVALID_ARGUMENT.LibraSystem::ENOT_AN_ACTIVE_VALIDATOR`
+    * Update is called too soon (updates from operators are rate-limited - two updates should be at least 5 minutes apart): `LibraSystem::ECONFIG_UPDATE_RATE_LIMITED`
+
+The following script relevant to validator's operations can also be run by any LibraAccount's owner:
+* [`rotate_authentication_key`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/rotate_authentication_key.move) - to rotate the authentication key of the Validator Owner Account
+  * Parameters:
+    * `new_key: vector<u8>`
+  * Events: none
+  * Errors:
+    * Account does not exist: `Error::make(category = Errors::NOT_PUBLISHED, reason = LibraAccount::EACCOUNT)`
+    * Key rotation capability was delegated to another account: `Error::make(category = Errors::INVALID_STATE, reason = LibraAccount::EKEY_ROTATION_CAPABILITY_ALREADY_EXTRACTED)`
+    * Malformed authentication key: `Error::make(category = Errors::INVALID_ARGUMENT, reason = LibraAccount::EMALFORMED_AUTHENTICATION_KEY)`
+
+## Formal verification guarantees on the Move Implementation
+TODO: These may change as verification work proceeds.
```

Comment:
@DavidLDill, when do you think this could be wrapped up?

Reply:
Done here after talking to DavidDill.